relayer/application-standard.ts

```diff
@@ -1,31 +1,34 @@
-import { RelayerApp, RelayerAppOpts } from "./application";
+import { RelayerApp, RelayerAppOpts } from "./application.js";
```

Comment:
this is weird :thinking: 
Why do you have to add the file extension? why is it `.js` instead of `.ts`?

Reply:
this is the typescript way to work with modules: https://www.typescriptlang.org/docs/handbook/esm-node.html